modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml

```diff
@@ -13,6 +13,18 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	</properties>
 	<dependencies>
+#if (${dependencyManagementEnabled} == "true")
```

Comment:
The dependencies that are added if dependencyManagement is enabled should be exactly the same artifacts as in the 'else' section.